README.md

```diff
@@ -1604,6 +1606,30 @@ will be ignored.
 
 ### Utility functions
 
+#### pipe
+
+<details><summary><code>pipe :: Future a b ~> (Future a b -> c) -> c</code></summary>
+
+```hs
+pipe :: Future a b ~> (Future a b -> c) -> c
+```
+
+</details>
+
+A method available on all Futures to allow arbitrary functions to be included
+in a fluent-style method chain.
```

Comment:
I understand what you mean by _arbitrary functions_, but the wording is imprecise. You're referring to functions not defined by Fluture as arbitrary, but readers may interpret _arbitrary_ as synonymous with _any_. As the function must accept a future as input, _arbitrary_ is too broad, I think.

Reply:
I cannot quite think of a better word, what do you think of *arbitrary functions over Futures*?

Reply:
That's perfect!

Reply:
:zap: